test/class_refactor.md

```
@@ -0,0 +1,120 @@
+# Test Refactor Plan: Moving Shared Code from test_markdownify.js to test_shared.js
+
+## Overview
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)